tests/test_snapshot/test_raises.py

```diff
@@ -0,0 +1,27 @@
+""" Tests for snappiershot.snapshot._raises.py """
+
+import pytest
+from snappiershot.snapshot import Snapshot
+from snappiershot.snapshot._raises import _RaisesContext
+
+
+def test_no_exception_raised(snapshot: Snapshot):
+    """ Test that the expected error is raised when no exception is caught. """
+    # Arrange
+    raises = _RaisesContext(snapshot, ValueError, False)
```

Comment:
this `ValueError` is not tied to the exception type (also value error) in teh assert, correct? Or do I misunderstand how this test works?

Reply:
Ah yeah this misleading. I'll change this to some different error type